frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -40,15 +44,24 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
   const { ocsConfig, close, cancel } = props;
   const deviceSets: DeviceSet[] = ocsConfig?.spec.storageDeviceSets || [];
 
-  const [response, loadError, loading] = usePrometheusPoll(cephCapacityResource);
+  const [values, loading] = usePrometheusQueries(queries, parser as any);
+  const loadError = values.every((item) => typeof item === 'undefined' || item === 0);
+
+  const totalCapacityMetric = values?.[0];
+  const usedCapacityMetric = values?.[1];
+
+  // Adjusted units
+  const usedCapacityAdjusted = humanizeBinaryBytes(usedCapacityMetric);
+  const availableCapacityAdjusted = humanizeBinaryBytes(totalCapacityMetric - usedCapacityMetric);
```

Comment:
We just need to show used capacity out of total capacity.
Non need to calculate the available.

Reply:
Sorry, i removed this last time and tested, but i forgot to clear browser cache, so it was working fine last time. But today when i loaded, it was showing in bytes. So we need to keep this. It is converting bytes into required uints.